packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -119,6 +120,25 @@ const VideoGalleryStory = (args): JSX.Element => {
       };
     });
 
+  const localParticipant = MockLocalParticipant;
+  localParticipant.isScreenSharingOn = args.screenShareExperience === 'presenter';
+
+  if (remoteParticipants.length > 0) {
+    remoteParticipants[0].isScreenSharingOn = args.screenShareExperience === 'viewer';
+
+    if (args.screenShareExperience === 'viewer') {
+      const renderElement = document.createElement('img');
+      renderElement.src = 'images/hero.svg';
```

Comment:
nit: are there stock images of a windows desktop we could use instead here?

Reply:
Perhaps: https://microsoft.sharepoint.com/teams/BrandCentral/Pages/Bundles/WIN22_Windows365_Start_V2_3x2_en-US.aspx
(or something here https://microsoft.sharepoint.com/teams/BrandCentral/Search/Pages/BCGeneralResults.aspx?k=windows#anchorScreens)

Reply:
we don't have any in our images folder

Reply:
Yeah, you'll need to add one, I assume we are allowed to use the stock images on brand central:
![image](https://user-images.githubusercontent.com/2684369/142267018-34d96724-3877-4f73-9d76-84e0271d39dd.png)

cc @ddematheu2

Reply:
If not, here is a screencap of my desktop I give permission to use lol
![image](https://user-images.githubusercontent.com/2684369/142274949-1b55df82-8b82-4594-8210-109376e6a206.png)

Reply:
@JamesBurnside updated with 
![image](https://user-images.githubusercontent.com/82416644/142288311-2a82ac49-5216-47be-8dfd-c95c220bb551.png)